### PR TITLE
Add paragraph about needing privileges to create and drop indexes and constraints

### PR DESCRIPTION
**Pull request by Hunterness**

### CODE REVIEW

**Review by Lojjs — APPROVED** [review]
Looks good to me

File: cypher/cypher-docs/src/docs/dev/ql/administration/constraints/index.asciidoc [hunk 1]
```diff
@@ -166,6 +166,10 @@ IS NODE KEY
 | Drop a node key constraint without specifying a name.
 |===
 
+Creating a constraint requires <<administration-security-administration-database-constraints, the `CREATE CONSTRAINT` privilege>>,
+while dropping a constraint require <<administration-security-administration-database-constraints, the `DROP CONSTRAINT` privilege>>.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/administration/constraints/index.asciidoc]:
```suggestion
while dropping a constraint requires <<administration-security-administration-database-constraints, the `DROP CONSTRAINT` privilege>>.
```

File: cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc [hunk 2]
```diff
@@ -114,6 +114,10 @@ n.propertyName_n)
 | Drop a composite index without specifying a name. 
 |===
 
+Creating an index requires <<administration-security-administration-database-indexes, the `CREATE INDEX` privilege>>,
+while dropping an index require <<administration-security-administration-database-indexes, the `DROP INDEX` privilege>>.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc]:
```suggestion
while dropping an index requires <<administration-security-administration-database-indexes, the `DROP INDEX` privilege>>.
```

**Review by AlexicaWright — COMMENTED** [review]
Just two small grammar-errors, otherwise looks good 👍

**Pull request merged**